Flutter/calendar/builders.md

```diff
@@ -328,7 +328,7 @@ The [ResourceViewHeaderBuilder](https://pub.dev/documentation/syncfusion_flutter
 
 
 {% tabs %}
-{% highlight Dart %}
+{% highlight dart hl_lines="7 8" %}
```

Comment:
Highlight inside method also

https://help.syncfusion.com/flutter/calendar/callbacks#view-changed-callback
https://help.syncfusion.com/flutter/calendar/callbacks#calendar-tap-callback
https://help.syncfusion.com/flutter/calendar/callbacks#calendar-details-callback
https://help.syncfusion.com/flutter/calendar/callbacks#long-press-callback

Reply:
Highlighted the inside method .